packages/lib/server-only/admin/get-recipients-stats.ts

```diff
@@ -19,9 +19,10 @@ export const getRecipientsStats = async () => {
 
   results.forEach((result) => {
     const { readStatus, signingStatus, sendStatus, _count } = result;
-    stats[readStatus] += _count;
-    stats[signingStatus] += _count;
-    stats[sendStatus] += _count;
+    stats[readStatus as keyof typeof stats] += _count;
+    stats.TOTAL_RECIPIENTS += _count;
+    stats[signingStatus as keyof typeof stats] += _count;
+    stats[sendStatus as keyof typeof stats] += _count;
     stats.TOTAL_RECIPIENTS += _count;
```

Comment:
Did I accidentally do this to you?

Reply:
Looks like I did this to myself, I see a repetition and I'm going to remove that